src/components/MaterialUI/Inputs/DatePicker.js

```diff
@@ -125,13 +146,15 @@ const WrappedDatePicker = ({
 				<div className={classes.datePickerContainer}>
 					<DatePicker
 						{...props}
-						dateFormat={dateFormat || (useDate && useTime ? "P p" : !useDate && useTime ? "p" : "P")}
+						dateFormat={dateFormat || createFormat(useDate, useTime, selectedLocale)}
 						selected={startDate}
 						onChange={date => updateDate(date, metadata)}
 						showTimeInput={useTime ?? false}
 						useTime={useTime ?? false}
 						customTimeInput={
-							useTime ? <TimePicker showTimeZone={showTimeZone} requestedTimeZone={timePickerTimeZone} /> : null
+							useTime ? (
+								<TimePicker showTimeZone={showTimeZone} showAMPM={false} requestedTimeZone={timePickerTimeZone} />
```

Comment:
with showAMPM always "false" you will have timePicker always in 24h format. and now it changes just format of date in datePicker input 
![image](https://user-images.githubusercontent.com/22075747/124284499-7187de00-db23-11eb-923a-489ebbae2bda.png)